Packs/Accessdata/Integrations/AccessdataV2/AccessdataV2.yml

```
@@ -0,0 +1,242 @@
+commonfields:
+  id: AccessData
+  version: -1
+
+name: AccessData
+display: AccessData
+category: Forensics & Malware Analysis
+description: Use the AccessData package to integrate with the AccessData platform enabling the automation of case/evidence management and endpoint collection.
+
+configuration:
+- display: Web Protocol
+  name: PROTOCOL
+  defaultvalue: http
+  type: 0
+  required: true
+- display: Service URL (FQDN or IP Address.)
+  name: SERVER
+  defaultvalue: localhost
+  type: 0
+  required: true
+- display: Service Listening Port
+  name: PORT
+  defaultvalue: 4443
+  type: 0
+  required: true
+- display: "The API authentication key. http://support.accessdata.com/hc/en-us/articles/360053994573-Generating-API-Keys"
+  name: APIKEY
+  defaultvalue: 00000000-abcd-1234-5678-000000000000
+  type: 0
+  required: true
+- display: "The path to the public certificate required to authenticate."
+  name: PUBLIC_CERT
+  defaultvalue: 
+  type: 0
+  required: false
+
+script:
+  commands:
+
+  - name: accessdata-api-get-case-by-name
+    arguments:
+    - name: name
+      required: true
+      description: The name of the case.
+    outputs:
+    - contextPath: accessdata.case.id
+      description: The identifier of the case.
+      type: integer
+    - contextPath: accessdata.case.name
+      description: The name of the case.
+      type: string
+    - contextPath: accessdata.case.casefolder
+      description: The path to the case data.
+      type: string
+    description: Returns a case's information determined by matching names.
+
+  - name: accessdata-api-create-case
+    arguments:
+    - name: name
+      required: true
+      description: The name of the case.
+    outputs:
+    - contextPath: accessdata.case.id
+      description: The identifier of the case.
+      type: number
+    - contextPath: accessdata.case.name
+      description: The name of the case.
+      type: string
+    - contextPath: accessdata.case.casefolder
+      description: The path to the case data.
+      type: string
```

Please change context to be same as `outputs_prefix` in `CommandResults`: 
```suggestion
    - contextPath: Accessdata.Case.ID
      description: The identifier of the case.
      type: integer
    - contextPath: Accessdata.Case.Name
      description: The name of the case.
      type: string
    - contextPath: Accessdata.Case.CaseFolder
      description: The path to the case data.
      type: string
```